pywemo/subscribe.py

```diff
@@ -91,6 +221,11 @@ def sleep(secs):
         self._httpd = None
         self._port = None
 
+    @property
+    def port(self) -> int:
```

Comment:
I like having this property, but this class does already have the `self._port` variable, right?  Should it be removed in favor of this?

Reply:
Good call. I removed `_port` and restructured things a bit.